Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM 018923174646.dkr.ecr.us-west-2.amazonaws.com/hls-base-3.5.1
+FROM 018923174646.dkr.ecr.us-west-2.amazonaws.com/hls-base:v3.5.2
```

Comment:
`hls-base-3.5.1` was last updated on Oct 13, 2023 and there exists as `hls-base@v3.5.1` from Jan 22, 2025 (from the HLS-VI release)

I'm assuming this difference stems from the `hls-landsat` container not needing any updates for a long while